src/Calibration/Calibration.cpp

```diff
@@ -62,8 +62,16 @@ namespace ZividPython::Calibration
                 },
                 py::arg("camera"),
                 py::arg("settings_2d"))
-            .def("estimate_intrinsics", [](ReleasableFrame &releasableFrame) {
-                return Zivid::Experimental::Calibration::estimateIntrinsics(releasableFrame.impl());
-            });
+            .def("estimate_intrinsics",
+                 [](ReleasableFrame &releasableFrame) {
+                     return Zivid::Experimental::Calibration::estimateIntrinsics(releasableFrame.impl());
+                 })
+            .def(
+                "pixel_mapping",
+                [](ReleasableCamera &releasableCamera, const Zivid::Settings &settings) {
+                    return Zivid::Experimental::Calibration::pixelMapping(releasableCamera.impl(), settings);
+                },
+                py::arg("camera"),
+                py::arg("settings"));
```

Comment:
This is experimental in C++, and thus should really be in a experimental namespace in Python. It seems we have done a bad mistake for the other ones abvove here by not separating Experimental to a separate namespace/module.

Reply:
Hmm I guess this is internal code only and that the public part is infact experimental. Then it looks Ok

Reply:
Yeah this is how we've done it before, with for example infield correction.